### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by Jon-MarkHampson**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -79,43 +93,82 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+float restaurantBillShare(int numberOfDiners =12, float foodCost = 1234.56f, float drinkCost = 569.99f, bool serviceGood = false, char country = 'e', double acceptableTip = 0.075)
+{
+    ignoreUnused(numberOfDiners, foodCost, drinkCost, serviceGood, country, acceptableTip);
+    return {};
+}
 /*
  5)
  */
-
+double stressLevel(bool doSport = false, int numberOfChildren = 5, bool didSchoolRun = true, double income = 45678.23, bool happyWithJob = false, int numberOfFriends = 4)
```
Comment (matkatmusic) [inline main.cpp]: Add a verb to these function names
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -79,43 +93,82 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+{
+    ignoreUnused(fishCaught, totalWeightCaught, biggestCatchOfTheDay, hoursFishing);
+    return {};
+}
 /*
  8)
  */
-
+void babyBedTime(float timeInbath, bool pyjamasOn = true, int numberOfStoires = 3, bool nightLightOn = true)
+{
```
Comment (matkatmusic) [inline main.cpp]: Add a verb to this function names
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -79,43 +93,82 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(fishWeight, numberOfChips, salt, vinegarInMl, mushyPeas);
+    return {};
+}
 /*
  10)
- */
-
+ */
+float jewelleryValue(int numberOfRing = 5, int numberOfNecklaces = 3, int numberOfBraclets = 1, bool gold = true, bool silver = false, float totalWeightRings = 34.56f, float totalWeightNecklaces = 11.11f, float totalWeightBraclets = 9.87f)
+{
```
Comment (matkatmusic) [inline main.cpp]: Add a verb to this function names
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Some of your functions still don't conform to the guideline of Name things what they are or what they do
It's very important to choose great names for your functions and variables.
you want your code to read as though you're reading instructions from a children's toy manual.
    processOutputGain(buffer);
    
    processOutputMeters(buffer);
    

    if( analyzerIsEnabled && !analyzerIsPreEq )
    {
        leftChannelSampleFifo.update(buffer);
        rightChannelSampleFifo.update(buffer);
    }
Without even knowing how these functions are implemented, can you infer what they do, based on the function names and variable names used?

You should be using this idea to pick names for your functions that include verbs in the name.
- [x] revise

**Comment by Jon-MarkHampson** [comment]
requested revisions completed

File: main.cpp [hunk 4]
```diff
@@ -79,43 +93,82 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+float timeRunningMarathon(int numberOfTrianingSessions, float numberOfTrainingMiles = 523.87f, bool newRunningShoes = true)
+{
```
Comment (matkatmusic) [inline main.cpp]: This function name has no verb in it.
the first function argument is misspelled.
- [x] revise

File: main.cpp [hunk 5]
```diff
@@ -79,43 +93,82 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    return {};
+}
 /*
  8)
  */
-
+bool IsTheBabyAsleep(float timeInbath, bool pyjamasOn = true, int numberOfStoires = 3, bool nightLightOn = true)
+{
+    ignoreUnused(timeInbath, pyjamasOn, numberOfStoires, nightLightOn);
+}
```
Comment (matkatmusic) [inline main.cpp]: This function doesn't return a value and it is declared that it should.
The function name doesn't conform to the course coding standard.
- [x] revise

File: main.cpp [hunk 6]
```diff
@@ -79,43 +93,82 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(fishWeight, numberOfChips, salt, vinegarInMl, mushyPeas);
+    return {};
+}
 /*
  10)
- */
-
+ */
+float CalculateJewelleryValue(int numberOfRing = 5, int numberOfNecklaces = 3, int numberOfBraclets = 1, bool gold = true, bool silver = false, float totalWeightRings = 34.56f, float totalWeightNecklaces = 11.11f, float totalWeightBraclets = 9.87f)
+{
```
Comment (matkatmusic) [inline main.cpp]: The function name doesn't conform to the course coding standard.
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Your project produces warnings and crashes when compiled and executed.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**